### PR TITLE
Versioning when publishing

### PR DESCRIPTION
**Pull request by damnhipster**

### CODE REVIEW

File: src/core/publish.js [hunk 1]
```diff
@@ -10,16 +10,25 @@ var publisher = new Publisher({
   timeout: 			    config.get('aws.s3.timeout')
 });
 
+function addToFileName(name, part) {
```
Comment (bneiluj) [inline src/core/publish.js]: I think it should be part of `utils.js` file.

Comment (damnhipster) [inline src/core/publish.js]: It's not the most concise bit of code, but okay.

Comment (damnhipster) [inline src/core/publish.js]: Done

**Comment by bneiluj** [comment]
happy to merge the branch when travis gets ready.

**Pull request merged**